.github/workflows/forest.yml

```diff
@@ -119,8 +119,21 @@ jobs:
       - name: Set permissions
         run: |
           chmod +x ~/.cargo/bin/forest*
+      - uses: actions/setup-go@v4
+        with:
+          go-version: "^1.20"
+      - name: Install go-car
+        run: go install github.com/ipld/go-car/cmd/car@v0.0.0-20230704002448-4de031a1d8a8
+        env:
+          CC: ""
+          CXX: ""
       - name: Other commands check
         run: ./scripts/tests/calibnet_other_check.sh
+      - name: Verify saved state root .car file
+        run: |
+          # File name is hard-coded in ./scripts/tests/calibnet_other_check.sh
+          car verify bafy2bzacedjq7lc42qhlk2iymcpjlanntyzdupc3ckg66gkca6plfjs5m7euo.car
+          car inspect bafy2bzacedjq7lc42qhlk2iymcpjlanntyzdupc3ckg66gkca6plfjs5m7euo.car
```

Comment:
What's the value of `car inspect`? Can it fail even after `car verify` has succeeded? If it's only used for debugging then let's remove it.

Reply:
Removed